crates/core/tedge_actors/src/builders.rs

```diff
@@ -97,15 +97,14 @@ pub struct NoConfig;
 ///
 /// An actor whose builder is a `MessageSink<M, C>` can be connected to any other actor
 /// whose builder is a `MessageSource<M, C>` so that the sink can receive messages from that source.
-///
-/// A sink might be interested only in a subset of the messages emitted by the source.
-/// For that purpose each source implementation defines a `Config` type parameter,
-/// and the sink has to provide the configuration value specific to its needs.
-pub trait MessageSink<M: Message, Config> {
+pub trait MessageSink<M: Message> {
```

Comment:
The `Config` type parameter is now useless (all the implementations were using `NoConfig`.